### PR TITLE
Upgrade 'canvas'

### PR DESCRIPTION
**Pull request by MikuroXina**
- Upgrade 'canvas' to 2.5.0
- Add yarn.lock
- Format
- Update this version to 0.0.3

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -37,29 +37,31 @@ const parseArgs = args => {
  *      Ignore other all options ignored when args specified.
  */
 module.exports = function deream(options = {}) {
-    const _options = Object.assign({
-        args: null,
-        dest: null,
-    }, options);
+  if (options.args === null || options.args === undefined) {
```
Comment (hanakla) [inline index.js]: You can do this

```suggestion
  if (options.args == null) {
```

File: index.js [hunk 2]
```diff
@@ -37,29 +37,31 @@ const parseArgs = args => {
...
+  }
 
-    if (options.args === null) {
-        throw new Error('options.args must be specified');
-    }
-
-    if (options.dest === null) {
-        throw new Error('options.args must be specified');
-    }
+  if (options.dest === null || options.dest === undefined) {
```
Comment (hanakla) [inline index.js]:
```suggestion
  if (options.dest == null) {
```

**Review by hanakla — CHANGES_REQUESTED** [review]
Reviewed, Thanks for your PR!

**Comment by MikuroXina** [comment]
Sorry, I think better that made the function using default values so I added 3e947f3 .

**Comment by MikuroXina** [comment]
I aborted this because replied nothing yet.

**Pull request closed without merging**